resolwe/flow/utils/test.py

```diff
@@ -224,6 +224,8 @@ def tearDown(self):
                 shutil.rmtree(data_dir, ignore_errors=True)
                 shutil.rmtree(export_dir, ignore_errors=True)
 
+        Process.objects.all().delete()
```

Comment:
This is indeed strange. Where did this contributor with id `30` come from? Is this the value of `self.admin.pk` in ProcessTestCase?